static/scripts/socketio.js

```diff
@@ -0,0 +1,75 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
you could move your script import at the bottom of the body or add 'defer' as a script property - <script defer></script>